src/hooks/useAntTheme/useAntTheme.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable func-names */
```

Comment:
why these disabling? can you follow the lint rules (or update them to better suite our needs)?

Reply:
Done, and corrected `useTheme.test.tsx` too

Reply:
thanks, didn't notice it was implemented this way also for the other file 🤦